### PR TITLE
Add `AggregateAndProof` to protobuf

### PR DESCRIPTION
**Pull request by terencechain**
part of #3835 

Added a protobuf message for `AggregateAndProof` used for naive aggregation strategy

### CODE REVIEW

File: proto/beacon/p2p/v1/types.proto [hunk 1]
```diff
@@ -101,3 +101,12 @@ message CompactCommittee {
   // The list of the validator indices in the committee.
   repeated uint64 compact_validators = 2 [(gogoproto.moretags) = "ssz-max:\"4096\""];
 }
+
+message AggregateAndProof {
```
Comment (prestonvanloon) [inline proto/beacon/p2p/v1/types.proto]: Is this a p2p message?

**Comment by prestonvanloon** [comment]
Ping on my previous comment, is this a p2p message?

File: proto/beacon/p2p/v1/types.proto (hunk 1 above)
Comment (terencechain) [reply proto/beacon/p2p/v1/types.proto]: Yes. It is a p2p message

**Comment by terencechain** [comment]
Closing this. Will open another one to target v0.9

**Pull request closed without merging**